core/chaincode/lifecycle/lifecycle.go

```diff
@@ -423,7 +441,7 @@ func (ef *ExternalFunctions) ApproveChaincodeDefinitionForOrg(chname, ccname str
 		return errors.WithMessage(err, "could not serialize chaincode package info to state")
 	}
 
-	logger.Infof("successfully approved definition %s, name '%s' on channel '%s'", cd, ccname, chname)
+	logger.Infof("Successfully approved definition %s, name '%s' on channel '%s'", cd, ccname, chname)
```

Comment:
why did we change this message? Historically, we haven't tried to convert our messages to lower case, only errors.

Reply:
it's changing from lower case to upper case, to align with other log messages.

Reply:
This file is odd. We have many errors that are upper case but our messages are lower case. Doesn't make sense. In most of fabric, our info messages are upper case.

Doesn't hold this up but we should probably revisit errors/messages in here as a separate item.

Reply:
will do a follow-up